colors/onedark.vim

```diff
@@ -259,6 +259,9 @@ call s:h("Visual", { "fg": s:visual_black, "bg": s:visual_grey }) " Visual mode
 call s:h("VisualNOS", { "bg": s:visual_grey }) " Visual mode selection when vim is "Not Owning the Selection". Only X11 Gui's gui-x11 and xterm-clipboard supports this.
 call s:h("WarningMsg", { "fg": s:yellow }) " warning messages
 call s:h("WildMenu", { "fg": s:black, "bg": s:blue }) " current match in 'wildmenu' completion
+if get(g:, 'onedark_hide_endofbuffer', 1)
```

Comment:
```suggestion
" If enabled, will style end-of-buffer filler lines (~) to appear to be hidden.
if get(g:, 'onedark_hide_endofbuffer', 0)
```

Reply:
I'd move this entire block to be right after the `DiffText` highlighting since `EndOfBuffer` appears directly after `DiffText` in `:h highlight-groups`.

Reply:
One other important change was that the default value should be `0` if the setting isn't configured, so the default styling stays as it is.